main.js

```diff
@@ -9,4 +9,271 @@ hamburger.addEventListener('click', () => {
+    featuredImage: 'img/work1.png',
+    author: 'Canopy',
+    type: 'Back End Dev',
+    year: '2022',
+    tech: ['Codekit', 'GitHub', 'Javascript'],
+    liveLink: 'https://htetnaing0814.github.io/Portfolio/',
+    sourceLink: 'https://github.com/HtetNaing0814/Portfolio',
+  },
+
+  {
+
+    name: 'Multi-Post Stories',
+    description: "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    featuredImage: 'img/work2.png',
+    author: 'Canopy',
+    type: 'Back End Dev',
+    year: '2015',
+    tech: ['html', 'css', 'Javascript'],
+    liveLink: 'https://htetnaing0814.github.io/Portfolio/',
+    sourceLink: 'https://github.com/HtetNaing0814/Portfolio',
+  },
+
+  {
+    name: 'Tonic ',
+    description: "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    featuredImage: 'img/work3.png',
+    author: 'Canopy',
+    type: 'Back End Dev',
+    year: '2015',
+    tech: ['html', 'css', 'Javascript'],
+    liveLink: 'https://htetnaing0814.github.io/Portfolio/',
+    sourceLink: 'https://github.com/HtetNaing0814/Portfolio',
+  },
+
+  {
+    name: 'Tonic ',
+    description: "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    featuredImage: 'img/work4.png',
+    author: 'Canopy',
+    type: 'Back End Dev',
+    year: '2015',
+    tech: ['html', 'css', 'Javascript'],
+    liveLink: 'https://htetnaing0814.github.io/Portfolio/',
+    sourceLink: 'https://github.com/HtetNaing0814/Portfolio',
+  },
+
+];
+
+for (let i = 0; i < items.length; i += 1) {
+  let li = '';
+  for (let k = 0; k < items[i].tech.length; k += 1) {
+    li += `<li>${items[i].tech[k]}</li>`;
+  }
+  if (i % 2 === 1) {
+    const data = document.createElement('div');
+    data.classList.add('work-items');
+    data.innerHTML = `
+         <div class="work reverse">
+            <div class="image">
+              <img src="${items[i].featuredImage}" alt=" " /> 
+            </div>
+            <div class="details">
+                    <h3>
+                    ${items[i].name}
+                    </h3>
+                <div class="w-description">
+                <h4>${items[i].author}</h4>
+                <ul>
+                  <li>${items[i].type}</li>
+                  <li>${items[i].year}</li>
+                </ul>
+                </div>
+                <div class="read-more">
+                <p>
+                 ${items[i].description}
+                </p>
+              </div>
+              <div class="tech-used">
+                 <ul>
+                    ${li}
+                  </ul>
+              </div>
+                <div class="button">
+                <button id="${items[i].id}"class="work-btn" type='button' onclick=test() >See Project</button>
+                </div>
```

Comment:
- [ ] When I click on the "See Project" button I get the error message: Uncaught ReferenceError: test is not defined
    at HTMLButtonElement.onclick. The popup is not appearing. Kindly fix that.

Reply:
Thank you @leonardodiasb  I unfortunately noticed it after my submission :(